README.rst

```diff
@@ -46,7 +46,26 @@ Usage
 Example 1: How do I download the datasets?
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-We have `plenty of them <https://github.com/okfn-brasil/serenata-de-amor/blob/master/research/CONTRIBUTING.md#datasets-data>`_ ready for you to download from our servers. And this toolbox helps you get them. Let's say you want your datasets at `data/`:
+We have `plenty of them <https://github.com/okfn-brasil/serenata-de-amor/blob/master/research/CONTRIBUTING.md#datasets-data>`_ ready for you to download from our servers. And this toolbox helps you get them. Here some examples:
+
+.. code-block:: bash
+
+    # without any arguments will download pre-propre-processedd datasets and store into export/ folder
+    $ serenata-toolbox
+
+    # will download these specific datasets and store into export/ folder
+    $ serenata-toolbox export --dataset federal-senate reimbursements
```

Comment:
Not clear what is happening here. Does `federal-senate reimbursements` stands for reimbursements from the Federal Senate, or data from the federal senate (which is called federal-senate.xz) as well as data from the chamber of deputies (which is called reimbursements-YYYY.xz)?

I would rather using the [module names](https://github.com/okfn-brasil/serenata-toolbox/tree/master/serenata_toolbox) for clarity – but this implementation raises an interesting issue on renaming the generated dataset to avoid this confusion, right? Maybe in another issue/PR ; ) 

Also, what is the role of export in this command? Is this the directory where data is to be saved, or a proper command?

Reply:
Ok, changing to modules. And yes, `export` is the example for output directory.

Reply:
> And yes, `export` is the example for output directory.

Just an idea; I'd add something with slashes to be clearer (maybe Rosie's default)

```sh
$ serenata-toolbox /tmp/serenata-data --dataset federal-senate chamber-of-deputies
```